timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md

```diff
@@ -0,0 +1,284 @@
+# Explore stock market data
+
+When you've successfully collected 1-min intraday stock data, it's time to have some fun and explore the 
+data.
+
+Because of the high granularity of the dataset, there are numerous ways to explore it. For example, you could analyze stock prices and volumes on a minute-by-minute basis. With TimescaleDB, you could also bucket records into custom intervals (e.g. 2-min or 15-min) using TimescaleDB aggregate functions. 
+
+Let's see how it's done!
+
+## Install Plotly and Pandas
+
+To get started with data exploration, you need to install a couple of tools first:
+
+* Pandas, to query and structure the data
+* Plotly, to create visualizations quickly
+
+**Install both**
+```bash
+pip install plotly pandas
+```
+
+When you have those installed, you need to open a new Python file, or use a Jupyter notebook to 
+start exploring the dataset.
+
+## Establish database connection
+
+Use the configuration file you created earlier with `psycopg2` to create a database connection object:
+
+```python
+import config, psycopg2
+conn = psycopg2.connect(database=config.DB_NAME,
+                        host=config.DB_HOST,
+                        user=config.DB_USER,
+                        password=config.DB_PASS,
+                        port=config.DB_PORT)
+```
+
+In each data exploration script below, you need to reference this connection object to be able to 
+query the database.
+
+## Generate stock market insights
+
+Let's start off analyzing trading volumes, then have a look at weekly price points, and finally, dig deep on 
+price changes.
+
+<highlight type="tip">
+Let these queries serve as inspiration to you, and feel free to change things up, like the analyzed *time frame*, the *symbol* or other parts of the query. Have fun!
+</highlight>
+
+1. Which symbols have the highest transaction volumes?
+2. How did Apple's trading volume change over time?
+3. How did Apple's stock price change over time?
+4. Which symbols had the highest weekly gains?
+5. Weekly FAANG prices over time?
+6. Weekly price changes of Apple, Facebook, Google?
+7. Distribution of daily price changes of Amazon and Zoom
+8.  Apple 15-min candlestick chart
+
+### 1. Which symbols have the highest transaction volumes?
+
+Let's generate a bar chart that shows the most traded symbols in the last 14 days:
+
+```python
+query = """
+    SELECT symbol, sum(trading_volume) AS volume
+    FROM stocks_intraday
+    WHERE time < now() - INTERVAL '{bucket}'
+    GROUP BY symbol
+    ORDER BY volume DESC
+    LIMIT 5
+""".format(bucket="14 day")
+df = pd.read_sql(query, conn)
+fig = px.bar(df, x='symbol', y='volume', title="Most traded symbols in the last 14 days")
+fig.show()
+```
+
+![most traded symbols](https://assets.timescale.com/docs/images/tutorials/intraday-stock-analysis/most_traded_symbols.png)
```

Comment:
Broken link to me.

Reply:
Looks like the actual rendered link is different than this one: https://assets.timescale.com/images/tutorials/intraday-stock-analysis/most_traded_symbols.png